Failed Test: test_history.py::test_run_without_data

Returned Stack Trace for test
```
____________________________ test_run_without_data _____________________________

mock_telebot = <MagicMock name='telebot' id='139784577395280'>
mocker = <pytest_mock.plugin.MockerFixture object at 0x7f22222ab1d0>

    @patch("telebot.telebot")
    def test_run_without_data(mock_telebot, mocker):
        MOCK_USER_DATA = test_read_json()
        mocker.patch.object(history, "helper")
        history.helper.getUserHistory.return_value = MOCK_USER_DATA["1574038305"]
        MOCK_Message_data = create_message("Hello")
        mc = mock_telebot.return_value
        mc.reply_to.return_value = True
        history.run(MOCK_Message_data, mc)
>       assert mc.reply_to.called
E       AssertionError: assert False
E        +  where False = <MagicMock name='telebot().reply_to' id='139784580959888'>.called
E        +    where <MagicMock name='telebot().reply_to' id='139784580959888'> = <MagicMock name='telebot()' id='139784580158736'>.reply_to

test/test_history.py:77: AssertionError
```

--- a/test/test_history.py
+++ b/test/test_history.py
@@ -74,7 +74,7 @@ def test_run_without_data(mock_telebot, mocker):
     mc = mock_telebot.return_value
     mc.reply_to.return_value = True
     history.run(MOCK_Message_data, mc)
-    assert mc.reply_to.called
+    assert mc.send_message.called
 
 
 @patch("telebot.telebot")

